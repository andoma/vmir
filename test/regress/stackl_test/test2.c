// test if/else

int main()
{
    if (1)
        printf("if works\n");
    else
        printf("if 1 broken\n");

    if (0)
        printf("if 2 broken\n");
    else
        printf("if-else works\n");

    return 0;
}

