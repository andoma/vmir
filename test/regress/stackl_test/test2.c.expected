if works
if-else works
